Common/cpp/ReanimatedRuntime/RNRuntimeDecorator.cpp

```diff
@@ -32,7 +30,7 @@ void RNRuntimeDecorator::decorate(
   rnRuntime.global().setProperty(rnRuntime, "_IS_FABRIC", isFabric);
 
 #ifndef NDEBUG
-  checkJSVersion(rnRuntime);
+  checkJSVersion(rnRuntime, nativeReanimatedModule->getJSLogger());
```

Comment:
Previously it worked without passing `jsLogger`, why do we need it now?

Reply:
Previously we were throwing runtime errors there, now we log stuff.

Reply:
Does it make sense to delegate `checkJSVersion` to `NativeReanimatedModule`?

Reply:
Does it make sense to delegate `checkJSVersion` to `NativeReanimatedModule`?